src/main/cli/cli.c

```diff
@@ -3556,6 +3557,43 @@ static void cliGpsPassthrough(const char *cmdName, char *cmdline)
 
     gpsEnablePassthrough(cliPort);
 }
+
+static void cliGpsGetConfig(const char *cmdName, char *cmdLine) {
+    UNUSED(cmdName);
+    UNUSED(cmdLine);
+
+    cliPrintLinef("# MON-VER acquired: %s", gpsData.acquiredMonVer ? "true" : "false");
+
+    if (gpsData.acquiredMonVer) {
+
+        cliPrintLinef("swVersion: ");
+        cliPrint(" ");
+        cliPrintLine(&gpsData.monVer.swVersion[0]);
+
+        cliPrintLinef("hwVersion: ");
+        cliPrint(" ");
+        cliPrint(&gpsData.monVer.hwVersion[0]);
+        cliPrint(" (");
+        cliPrint(ubloxVersionToString(ubloxDetectVersion(gpsData.monVer.hwVersion, sizeof(gpsData.monVer.hwVersion))));
+        cliPrintLine(")");
+
+        cliPrintLinef("extension: ");
+
+        for (size_t i = 0; i < 10; ++i) {
+            cliPrint(" ");
+            for(size_t j = i * 30; j < i * 30 + 30; ++j) {
+                if (j == i * 30 && gpsData.monVer.extension[j] == 0) {
```

Comment:
```suggestion
            size_t offset = i * 30;
            for (size_t j = offset; j < offset + 30; ++j) {
                if (j == offset && gpsData.monVer.extension[j] == 0) {
```

Reply:
just for readability?

Reply:
At least the `for (size_t` - missing space - other part is not really in a cli status request :)

Reply:
![image](https://github.com/betaflight/betaflight/assets/8344830/ed8e912d-422a-410e-a5e2-5f79eaa46f64)